met-web/src/components/common/Modals/EmailModal.tsx

```diff
@@ -0,0 +1,167 @@
+                        container
+                        direction="row"
+                        width="100%"
+                        xs={12}
+                        alignItems="flex-start"
+                        justifyContent="flex-start"
+                        rowSpacing={1}
+                    >
+                        <Grid item xs={12}>
+                            <FormControl required error={emailFormError.terms} component="fieldset" variant="standard">
+                                <FormControlLabel
+                                    control={
+                                        <Checkbox
+                                            onChange={(event) => {
+                                                setChecked(event.target.checked);
+                                                setEmailFormError({ ...emailFormError, terms: false });
+                                            }}
+                                        />
+                                    }
+                                    label="I agree to the terms and conditions above."
+                                />
+                                <FormHelperText>
+                                    {emailFormError.terms ? 'Please accept the terms and conditions' : ''}
+                                </FormHelperText>
+                            </FormControl>
+                        </Grid>
+                        <Grid item xs={12}>
+                            <MetLabel>Email Address</MetLabel>
+                            <TextField
+                                onChange={(e) => {
+                                    updateEmail(e.target.value);
+                                    setEmailFormError({ ...emailFormError, email: false });
+                                }}
+                                label=" "
+                                InputLabelProps={{
+                                    shrink: false,
+                                }}
+                                variant="outlined"
+                                error={emailFormError.email}
+                                helperText={emailFormError.email ? 'Please enter an email' : ''}
+                                fullWidth
+                            />
+                        </Grid>
+                    </Grid>
+
+                    <Grid
+                        item
+                        container
+                        xs={12}
+                        direction="row"
+                        justifyContent="flex-end"
+                        spacing={1}
+                        sx={{ mt: '1em' }}
+                    >
+                        <Stack
+                            direction={{ xs: 'column', sm: 'row' }}
+                            spacing={1}
+                            width="100%"
+                            justifyContent="flex-end"
+                        >
+                            {isSmallScreen ? (
```

Comment:
![image](https://user-images.githubusercontent.com/91914654/211934643-a5b8191f-bd8f-422f-82b9-e3aafef3a179.png)

Reply:
try doing this the above way, creds to @tiagograf-AOT

Reply:
thank you